scripts/Dockerfile.test

```diff
@@ -0,0 +1,43 @@
+# Container image for running the Ceph-CSI tests
```

Comment:
how about having a single dockerfile with multistage build support for container build and test. i see the way going forward to have everything inside one container with a single command

Reply:
That is quite difficult. I tried to use the `ceph/cecp:latest` image as base, but the tools we use for testing are not readily available on CentOS, and that is the target OS for the final images. Developers also often have a more modern distribution than the OS used for production, so I think there is some value in running tests with two distributions.

Reply:
but if we use fedora as the first base image and we can switch to ceph/ceph:latest later in the multistage docker build, its possible isn't it?

Reply:
It is possible, but we may have different versions of dependencies (`librados`, `librbd` and `libcephfs`). If the version difference is large enough, there could be problems during runtime. Compiling & linking against the versions that are available in the image will prevent potential issues.